be/src/io/fs/hdfs_file_writer.h

```diff
@@ -48,13 +51,32 @@ class HdfsFileWriter final : public FileWriter {
     bool closed() const override { return _closed; }
 
 private:
+    // Flush buffer into file cache, **Notice**: this would clear the underlying buffer
```

Comment:
```suggestion
    // Flush buffered data into HDFS client and write local file cache if enabled, **Notice**: this would clear the underlying buffer
```